DarkVectorTemplate.php

```diff
@@ -296,10 +296,10 @@ protected function renderPortal( $name, $content, $msg = null, $hook = null ) {
 			$msg = $name;
 		}
 		$msgObj = wfMessage( $msg );
-		$labelId = Sanitizer::escapeId( "p-$name-label" );
+		$labelId = Sanitizer::escapeIdForAttribute( "p-$name-label" );
 		?>
```

Comment:
This is correct but see L62 for another usage.

Reply:
Fixed